psdb/generateDataPSDB.js

```diff
@@ -0,0 +1,83 @@
+const faker = require('faker');
+
+// *** Fake Hours Data Generator to make restaurant hours more realistic *** //
+const makeFakeHoursData = () => {
```

Comment:
This function is doing 2 separate things, maybe see how you could break this up to be more modular. It looks to me like the function is:
- Creating the daily hours for each day of the week
- Converting those daily hours into strings for each day of the week

Both things are related, so I understand the idea of putting it all together, but maybe break out those 2 pieces into helper functions that are called the necessary amount of times in the larger makeFakeHoursData function. [This README](https://github.com/ryanmcdermott/clean-code-javascript#functions) is a great resource for best coding practices, and the section called "Functions should do one thing" would be a good thing to look over if you haven't already.

Reply:
Thanks for sharing that README, will be great for me to look through. Thanks for catching a good way to make the data generation more modular. I refactored accordingly.